firebase-ml-modeldownloader/src/main/java/com/google/firebase/ml/modeldownloader/FirebaseModelDownloaderRegistrar.java

```diff
@@ -45,15 +50,24 @@ public class FirebaseModelDownloaderRegistrar implements ComponentRegistrar {
   @NonNull
   @RequiresApi(api = VERSION_CODES.KITKAT)
   public List<Component<?>> getComponents() {
+    Qualified<Executor> bgExecutor = Qualified.qualified(Background.class, Executor.class);
+    Qualified<ExecutorService> bgExecutorService =
```

Comment:
is `ExecutorService` required here, can mlkit just use the plain `Executor` instead?